pulp_ansible/tests/functional/api/collection/test_consume_content.py

```diff
@@ -55,12 +57,15 @@ def test_consume_content(self):
         )
         self.addCleanup(self.client.delete, distribution["_href"])
 
+        collection_dir = tempfile.mkdtemp()
```

Comment:
We could add a clean up for this dir. Not a blocker.

Reply:
I figured since it was a temp dir, we wouldn't have to worry about it?

Reply:
Do we have to cleanup temp directories?

Reply:
The temp facilities will handle this for us is my understanding.